vite.config.ts

```diff
@@ -1,7 +1,9 @@
-import { defineConfig } from 'vite'
-import react from '@vitejs/plugin-react'
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
 
-// https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
-})
+  server: {
+    port: 3000,  // Змініть порт на 3000
```

Comment:
what the comment?Do u need it?
If u don't need it, delete all comments from the project